apps/web/lib/hooks/useSlots.ts

```diff
@@ -207,10 +209,13 @@ export const useSlots = (props: UseSlotsProps) => {
     return filteredTimes.map((time) => ({
       time,
       users: [user],
-      // Conditionally add the booking info slots object if there is already a booking during that time
+      // Conditionally add the attendees and booking id to slots object if there is already a booking during that time
       ...(currentSeats?.some((booking) => booking.startTime === time.toISOString()) && {
-        bookingInfo:
-          currentSeats[currentSeats.findIndex((booking) => booking.startTime === time.toISOString())],
+        attendees:
+          currentSeats[currentSeats.findIndex((booking) => booking.startTime === time.toISOString())]._count
+            .attendees,
+        bookingId:
+          currentSeats[currentSeats.findIndex((booking) => booking.startTime === time.toISOString())].id,
```

Comment:
Conditionally add bookingId and attendees if the time slot already has a booking with seats